qhub/template/stages/07-kubernetes-services/modules/kubernetes/services/kbatch/main.tf

```diff
@@ -0,0 +1,81 @@
+# Get this name dynamtically
+locals {
+  kbatch_service_account_name = "kbatch-kbatch-proxy"
+}
+
+resource "helm_release" "kbatch" {
+  name       = "kbatch"
+  namespace  = var.namespace
+  repository = "https://kbatch-dev.github.io/helm-chart"
+  chart      = "kbatch-proxy"
+  version    = "0.3.1"
+
+  values = concat([
+    file("${path.module}/values.yaml"),
+    jsonencode({
+      app = {
+        jupyterhub_api_token = var.jupyterhub_api_token
+        jupyterhub_api_url = "https://${var.external-url}/hub/api/"
+        extra_env = {
+          KBATCH_PREFIX = "/services/kbatch"
```

Comment:
What if we set this to `KBATCH_PREFIX = "/services/kbatch"`?  It would simplify the configure command slightly.
It would become 
`kbatch configure --kbatch-url="http://kbatch-kbatch-proxy.dev.svc.cluster.local" --token="<token>"`
instead of 
`kbatch configure --kbatch-url="http://kbatch-kbatch-proxy.dev.svc.cluster.local/services/kbatch" --token="<token>"`

Reply:
I think you're right, we don't need the `KBATCH_PREFIX`, especially if we don't plan on adding the url link (as discussed in the previous review comment). 

Removed.